server/src/controllers/notFound.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response } from 'express';
+
+const notFound = (req:Request, res:Response) => {
+  res.status(404).json('not found');
```

Comment:
It's good to return the status code in the response body
```suggestion
  res.status(404).json({ status: 404, msg: 'not found' });
```